packages/c/sshnpd/sshnpd.supp

```diff
@@ -0,0 +1,442 @@
+# Things to fix only main process
+{
+  FIXME cJSON_PrintUnformatted
+    Memcheck:Leak
+    match-leak-kinds: reachable
+    fun:realloc
+    fun:print
+    fun:cJSON_PrintUnformatted
+    fun:main
+}
+{
+  FIXME atclient_atsign_init
+    Memcheck:Leak
+    fun:malloc
+    fun:atclient_atsign_init
+}
```

Comment:
These also need to be unsuppressed & fixed to get rid of all pre-connection leaks